torch/nn/functional.py

```diff
@@ -2762,6 +2762,7 @@ def poisson_nll_loss(
     return ret
 
 
+# TODO: Pure Python impl - don't add a primTorch ref
```

Comment:
I would remove this comment -- it's not really a TODO since there's nothing to do